README.md

```diff
@@ -229,18 +231,19 @@ following debug messages in your log:
    method is applied for Clang-Tidy results too, because Clang-Tidy does not
    support bug identifier hash generation currently.
 
-
-Additional documentation
+  
+Developer Documentation
 ------------------------
-
-  * [User guide](docs/user_guide.md)
-  * [CodeChecker getting started](docs/usage.md)
   * [Requirements, external source dependencies](docs/deps.md)
   * [Architecture overview](docs/architecture.md)
   * [Package layout](docs/package_layout.md)
   * [Checker documentation](docs/checker_docs.md)
   * [Thrift interface](thrift_api/thrift_api.md)
```

Comment:
This .md file doesn't exist.

Reply:
This will be fixed in another pull request. #773